superset-frontend/src/features/alerts/AlertReportModal.tsx

```diff
@@ -385,47 +354,83 @@ interface NotificationMethodAddProps {
 }
 
 export const TRANSLATIONS = {
```

Comment:
This seems overkilling and making the file harder to read with a bunch of constants to be referring back to at the top of the file. What's the benefit of this VS just add the localized string inline?

Reply:
I agree, I was attempting to follow the precedents of the original file as closely as possible at the time. I'll change them into strings at their implementations.

Reply:
I left the titles and strings for validation errors, just in case they need to be edited before the final merge it could be easier to have them in one place